test/test_manifest.py

```diff
@@ -9,7 +9,8 @@
 import pytest
 
 import testutil
-from containerbuild import build_container_fixture  # pylint: disable=unused-import
+from containerbuild import \
```

Comment:
(just curious) why split this line? I really don't mind, is it for aesthetics or some editor defaults?

Reply:
oops.  That was `isort`.  I keep it on but I need to be careful when it changes things I'm not modifying.